content/sensu-go/5.0/reference/hooks.md

```diff
@@ -20,16 +20,18 @@ and based on the exit status code of that command (ex: `1`).
 Hook commands can optionally receive JSON serialized Sensu client data via
 STDIN.
 
+### Check response types
+
 Each **type** of response (ex: `non-zero`) can contain one or more hooks, and
 correspond to one or more exit status code. Hooks are executed, in order of
 precedence, based on their type:
 
-* `1` to `255`
-* `ok`
-* `warning`
-* `critical`
-* `unknown`
-* `non-zero`
+1. `1` to `255`
+2. `ok`
```

Comment:
If all of these options are labelled as `1.`, it _should_ be able to render the correct value.

```
 1. `1` to `255`
 1. `ok`
 1. `warning`
 1. `critical`
 1. `unknown`
 1. `non-zero`
```

Reply:
I believe you are correct! I'm going to hold off on this for now, but this is a great point and I'll definitely keep it in mind for future ordered lists. <3